### PR TITLE
[ci,rustfmt] Formatting due to rustfmt 0.99.1-stable in Rust 1.29.0.

### PR DESCRIPTION
**Pull request by fnichol**
https://blog.rust-lang.org/2018/09/13/Rust-1.29.html

Signed-off-by: Fletcher Nichol <fnichol@nichol.ca>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/core/src/os/process/windows_child.rs [hunk 1]
```diff
@@ -623,16 +623,19 @@ impl OpenOptions {
     }
 
     fn get_flags_and_attributes(&self) -> winapi::DWORD {
-        self.custom_flags | self.attributes | self.security_qos_flags
+        self.custom_flags
+            | self.attributes
+            | self.security_qos_flags
```
Comment (baumanj) [inline components/core/src/os/process/windows_child.rs]: Oooh, pretty

Comment (fnichol) [reply components/core/src/os/process/windows_child.rs]: Yeah, the choices being made are trending in a great direction

**Comment by fnichol** [comment]
Looks like our Travis workers are attempting to rebuild `libsodium` and are failing on an `HTTP/404` (this project moves its artifacts around rather aggressively). I'll see if I can fix with a split-out commit on this PR and resolve both...

**Pull request merged**